man/systemd-sysext.xml

```diff
@@ -467,6 +480,29 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>Configuration Files</title>
+
+    <para>Some options can be globally configured via configuration files. The
+    configuration files are always overridden by the command line options.</para>
```

Comment:
Just mention once that configuration files are overridden by env vars and options. This is completely expected.